NetcoinLib/BankSystem.cs

```diff
@@ -13,5 +19,21 @@ public BankSystem(INetcoinRepository netcoinRepository)
         public void ReadTextFile(string fileName) => _netcoinRepository.ReadSerializedData(fileName);
 
         public void SaveTextFile() => _netcoinRepository.Save();
+
+        public void WithdrawFromAccount(int accountId, decimal amountToWithdraw)
+        {
+            Account account = _netcoinRepository.GetAccounts().SingleOrDefault(x => x.AccountId == accountId);
+            if (account != null)
+            {
+                if (account.Balance >= amountToWithdraw)
+                    account.Balance = account.Balance - amountToWithdraw;
```

Comment:
What would happen if amountToWithdraw was for example -100?

Reply:
Uh oh